lib/npm_module.js

```diff
@@ -9,8 +10,28 @@ function npmModule(opts) {
 	if (this.description !== undefined)
 		this.summary = this.description.split('.')[0];
 	this.license = opts['license'];
-	this.project_url = opts['project_url'];
-	this.dependencies = opts['dependencies'];
+	this.project_url = opts['homepage'];
+	if (this.project_url !== undefined) {
+		this.project_url = opts['homepage'];
+	} else {
+		if (opts['bugs'] !== undefined) {
+			this.project_url = opts['bugs'].url;
+		} else if (opts['repository'] !== undefined) {
+			this.project_url = opts['repository'].url;
+		} else {
+			this.project_url = getHomepage(this.name);
```

Comment:
`this.name` is undefined here, it's set here https://github.com/dLobatog/npm2rpm/pull/14/files#diff-b9d2330ecffe80f21ea33ba7a5ac0942R41

You'd need to set the project_url after the name - and I'd recommend extracting that to a method